src/OmnichannelChatSDK.ts

```diff
@@ -1137,7 +1138,7 @@ class OmnichannelChatSDK {
             const {PreChatSurvey: preChatSurvey, msdyn_prechatenabled, msdyn_callingoptions, msdyn_conversationmode, msdyn_enablechatreconnect} = liveWSAndLiveChatEngJoin;
             const isPreChatEnabled = msdyn_prechatenabled === true || msdyn_prechatenabled == "true";
 
-            if (msdyn_enablechatreconnect && !this.isPersistentChat) { 
+            if ((msdyn_enablechatreconnect === true || msdyn_enablechatreconnect == "true") && !this.isPersistentChat) {
```

Comment:
Separate and move "isReconnectChatEnabled = msdyn_enablechatreconnect === true || msdyn_enablechatreconnect == "true"" below line 1138 for more readability?